prow/cluster/components/monitoring/prow_prometheus.yaml

```diff
@@ -32,8 +32,8 @@ spec:
     matchExpressions:
     - key: app
       operator: Exists
-  version: v2.7.1
-  baseImage: docker.io/prom/prometheus
+  version: v2.40.7
```

Comment:
The description refers to Prometheus 2.40.1. Please verify that it is correct

Reply:
I've updated the description, we're using version 2.40.7